libfwupdplugin/fu-fmap-firmware.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Benson Leung <bleung@chromium.org>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include "fu-firmware.h"
+
+#define FMAP_SIGNATURE		"__FMAP__"
+#define FMAP_VER_MAJOR		1	/* this header's FMAP major version */
+#define FMAP_VER_MINOR		1	/* this header's FMAP minor version */
+#define FMAP_STRLEN		32	/* maximum length for strings, */
+					/* including null-terminator */
+#define FMAP_AREANAME		"FMAP"
```

Comment:
This define only appears to be used in one source file, I think you should just move the define there.

Reply:
Dnoe.